src/prefect/logging/loggers.py

```diff
@@ -92,6 +93,12 @@ def get_run_logger(context: "RunContext" = None, **kwargs: str) -> logging.Logge
         logger = flow_run_logger(
             flow_run=flow_run_context.flow_run, flow=flow_run_context.flow, **kwargs
         )
+    elif (
+        get_logger("prefect.flow_run").disabled
+        and get_logger("prefect.task_run").disabled
+    ):
+        logger = logging.getLogger("null")
+        logger.addHandler(logging.NullHandler())
```

Comment:
This always retrieves the same logger so this will add a new handler each time this is called

Reply:
Do you think I should edit logging.yml or just set a NULL_LOGGER in code?

Reply:
Is there a reason we need to set a null handler on it? Can't we just return a logger that's not hooked up to anything?